coeus-impl/src/main/java/org/kuali/coeus/propdev/impl/budget/subaward/BudgetSubAwards.java

```diff
@@ -114,9 +113,6 @@
     @OneToMany(mappedBy = "budgetSubAward", orphanRemoval = true, cascade = { CascadeType.ALL })
     @OrderBy("budgetPeriod")
     private List<BudgetSubAwardPeriodDetail> budgetSubAwardPeriodDetails;
-
-    @OneToMany(mappedBy = "budgetSubAward", cascade = {CascadeType.REFRESH, CascadeType.REMOVE})
-    private List<BudgetLineItem> budgetLineItems;
     
```

Comment:
It's really too bad that we can't have JPA handle this.  Is this because our object graph isn't organized well.  It seems that JPA should be able to keep track of these instances when they are removed from other places.  Do you think it is because krad-data is doing something?